everything_shell/getenv.c

```diff
@@ -0,0 +1,53 @@
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+/**
+ * _getenv - gets an environment variable.
+ * @name: variable name.
+ *
+ * Return: returns pointer to value in environment or NUll if not found.
+ */
+char *_getenv(const char *name)
+{
+	extern char **environ;
+	char **environ_copy;
+	char *current_var;
+	unsigned int length = strlen(name);
+
+
+	environ_copy = environ;
```

Comment:
i actually asked chatgpt the same and it made me understand that since we are not performing any sort of write or execute commands, theres no need to duplicate or copy since we its just read only actions we are performing on the file, that is why F_OK was used on access.

Reply:
Why not use `environ` directly then?